src/platform_impl/web/event_loop/runner.rs

```diff
@@ -766,9 +764,7 @@ impl Shared {
         // * For each undropped `Window`:
         //     * The `register_redraw_request` closure.
         //     * The `destroy_fn` closure.
-        if self.0.event_loop_recreation.get() {
-            crate::event_loop::EventLoopBuilder::allow_event_loop_recreation();
-        }
+        crate::event_loop::EventLoopBuilder::allow_event_loop_recreation();
```

Comment:
We should mention in the changelog that event loop recreation is now always allows on Web.